core/src/main/res/values/arrays.xml

```diff
@@ -65,12 +65,14 @@
 
     <string-array name="feedNewEpisodesActionItems">
         <item>@string/global_default</item>
+        <item>@string/feed_new_episodes_action_add_to_queue</item>
         <item>@string/feed_new_episodes_action_add_to_inbox</item>
```

Comment:
I would move it below the inbox because the inbox is the default and should be on top.

Please also adapt the "global" array directly above this in the file.

Reply:
Ah, did not even notice the global array. Thanks, changed it now.